dora/core/client/fs/src/main/java/alluxio/client/file/ufs/DoraOutStream.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.client.file.ufs;
+
+import alluxio.AlluxioURI;
+import alluxio.client.file.FileOutStream;
+import alluxio.client.file.FileSystemContext;
+import alluxio.client.file.dora.DoraCacheClient;
+import alluxio.client.file.options.OutStreamOptions;
+import alluxio.grpc.CompleteFilePOptions;
+import alluxio.grpc.FileSystemMasterCommonPOptions;
+import alluxio.util.CommonUtils;
+
+import com.google.common.base.Preconditions;
+import com.google.common.io.Closer;
+
+import java.io.IOException;
+
+/**
+ * A Dora output stream.
+ */
+public class DoraOutStream extends FileOutStream {
+
+  /** Used to manage closeable resources. */
+  private final Closer mCloser;
+  private final FileSystemContext mContext;
+  private final OutStreamOptions mOptions;
+  private boolean mClosed;
+  protected final AlluxioURI mUri;
+  private final DoraCacheClient mDoraClient;
```

Comment:
nit: I think it might better to group these field as private final, private and then protected?

Reply:
Yes, you are right.